metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
@brandonmensing Thanks for chiming in here. We are not taking nanoseconds timestamp but the diffs, so the numbers "should" be a reasonable small integer. If we come to the conclusion that response_time in ns is nothing useful, should we then provide response_time at all in ms if it is often 0? It is still nice to see things that take too long. I'm still +1 on the ns approach but I can see the point that it is probably overkill. Make it an optional flag to enable?